static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-us-top-ticker.js

```diff
@@ -84,6 +33,7 @@ const noteFromJohn: AcquisitionsEpicTemplateCopy = {
     highlightedText:
         'Please make a year-end gift today to show your ongoing support for our independent journalism. Thank you.',
 };
+
 export const acquisitionsEpicUsTopTicker: EpicABTest = makeABTest({
```

Comment:
I think you may need to give the test a new name if you want people who were shown an unsuccessful variants to see the new variant

Reply:
Yeah have done that. Renamed the banner test which I converged to a single variant last week too, in case it suffers from the same problem (possibly not because some of the banner code goes straight to the mvt cookie). Deleted an old test. And put the switches to `SafeState = On` so we don't need to remember to switch them on after deploying